Project.toml

```diff
@@ -35,7 +35,7 @@ DocStringExtensions = "0.8, 0.9"
 Insolation = "0.8"
 JLD2 = "0.4"
 NCDatasets = "0.11, 0.12, 0.13"
-Plots = "1.39.0"
+Plots = "< 1.40.0"
```

Comment:
Ideally let's not merge this in. 

I'd rather temporarily disable CI on MacOS than pinning a package.